client/components/theme-upgrade-modal/index.tsx

```diff
@@ -222,7 +246,53 @@ export const ThemeUpgradeModal = ( {
 	};
 
 	const getBundledFirstPartyPurchaseModalData = (): UpgradeModalContent => {
-		const businessPlanPrice = businessPlanProduct?.combined_cost_display;
+		const getPlanText = ( planName: string, term: string, planPrice: string ) => {
+			switch ( term ) {
+				case 'three years':
+					return translate(
+						'Upgrade to a %(planName)s plan to select this theme and unlock all its features. It’s %(planPrice)s per three years with a 14-day money-back guarantee',
```

Comment:
Is there a compelling reason for this text to vary between plans? 🤔

Reply:
I don't think so, maybe we could get away with having only one translate right?

Reply:
I've just checked and it looks like it was designed this way (https://github.com/Automattic/wp-calypso/issues/70388) not so long ago :/